src/dlgTriggerEditor.cpp

```diff
@@ -6648,9 +6648,9 @@ void dlgTriggerEditor::autoSave()
     mpHost->saveProfile(QString(), qsl("autosave"));
 }
 
-void dlgTriggerEditor::enterEvent(TEnterEvent* pE)
+void dlgTriggerEditor::enterEvent(TEnterEvent* event)
```

Comment:
Because this argument is NOT used it is allowed to not give a name **at all** for it (then you do not need the `Q_UNUSED(event)` ***either***)! :astonished:

Reply:
Interesting feature but I think that would confuse more than help. Let's stay away from it and be explicit here.